util/jsonrpc-types/src/info.rs

```diff
@@ -55,7 +57,18 @@ pub struct DeploymentInfo {
     pub timeout: EpochNumber,
     /// specifies the epoch at which the softfork is allowed to become active.
     pub min_activation_epoch: EpochNumber,
-    /// With each epoch and softfork, we associate a deployment state. The possible states are
+    /// the length in epochs of the signalling period
+    pub period: EpochNumber,
+    /// the ratio of blocks with the version bit set required to activate the feature
+    pub threshold: Ratio,
+    /// The first epoch which the current state applies
+    pub since: EpochNumber,
```

Comment:
`struct DeploymentInfo` have `since` field, but this field doesn't appear on [rpc doc DeploymentInfo](https://github.com/nervosnetwork/ckb/tree/develop/rpc#type-deploymentinfo)

Reply:
I re-generated rpc doc in https://github.com/nervosnetwork/ckb/pull/3949